Assets/Scenes/Main.unity

```diff
@@ -26623,7 +26623,7 @@ Camera:
   far clip plane: 10000
   field of view: 60
   orthographic: 1
-  orthographic size: 1
+  orthographic size: 2.5
```

Comment:
my math is saying this should be 320, but that might be totally wrong in actuality lmao